doc/source/rclcpp_cpp_client_library_overview.rst

```diff
@@ -264,11 +269,11 @@ For example, the ``nav_msgs/msg/Path.msg`` Message has a list of "stamped" pose'
 Publishing with a ``Publisher``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In |rclcpp| publishing is achieved by creating a :cpp:class:`rclcpp::Publisher` class and calling :cpp:member:`rclcpp::Publisher::publish` with a Message as the first parameter.
+In |rclcpp| publishing is achieved by creating an :cpp:class:`rclcpp::Publisher` object and calling :cpp:member:`rclcpp::Publisher::publish` with a Message as the first parameter.
```

Comment:
The document had a mix. these changes were for consistency (but I'd pronounce it as `an rclcpp` so that'd get my vote)